solution-base/build.sh

```diff
@@ -22,18 +22,30 @@ DOCKER_SOCKET=${DOCKER_SOCKET:-unix:///var/run/docker.sock}
 HARDLINK=hardlink
 SKOPEO=skopeo
 SKOPEO_OPTS="--override-os linux --insecure-policy"
-OPERATOR_TAG=$(grep /operator: deps.txt | awk -F ':' '{print $2}')
+OPERATOR_TAG=$(grep kubedb/operator: deps.txt | awk -F ':' '{print $2}')
 SOLUTION_REGISTRY=metalk8s-registry-from-config.invalid/${PRODUCT_LOWERNAME}-${VERSION_FULL}
 
 KUBEDB_SCRIPT_BRANCH_TAG=89fab34cf2f5d9e0bcc3c2d5b0f0599f94ff0dca
 
+SOLUTION_ENV="zenko"
```

Comment:
I was using a replacable value here as I think the intention was to sed the value for whatever you choose as your metalk8s solution name. As we will not be able to define that here.

Reply:
you're correct; Forgotten that we had a conversation about this.